tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -59,7 +59,7 @@ Pull non-existent image
     Should contain  ${output}  image library/fakebadimage not found
 
 Pull image from non-existent repo
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull fakebadrepo.com:9999/ubuntu
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull fakebadrepo.com:9999/${ubuntu}
```

Comment:
this is pulling a non-existent image from a non-existent repo, no need to make this change here.